diff --git a/out/util/appFileCopier.js b/out/util/appFileCopier.js
index f762b47b85539eea2b38f6cf76692624a3b7e307..78b2f2ee9d55a972488536e23e752a5eeb7e246d 100644
--- a/out/util/appFileCopier.js
+++ b/out/util/appFileCopier.js
@@ -176,7 +176,7 @@ async function collectNodeModulesWithLogging(platformPackager) {
     const { tempDirManager, appDir, projectDir } = packager;
     let deps = undefined;
     const searchDirectories = Array.from(new Set([appDir, projectDir, await packager.getWorkspaceRoot()])).filter((it) => (0, builder_util_1.isEmptyOrSpaces)(it) === false);
-    const pmApproaches = [await packager.getPackageManager(), node_module_collector_1.PM.TRAVERSAL];
+    const pmApproaches = [node_module_collector_1.PM.NPM, await packager.getPackageManager(), node_module_collector_1.PM.TRAVERSAL];
     for (const pm of pmApproaches) {
         for (const dir of searchDirectories) {
             builder_util_1.log.info({ pm, searchDir: dir }, "searching for node modules");
